tools/jolie2java/src/joliex/java/impl/JavaDocumentCreator.java

```diff
@@ -979,4 +988,20 @@ private void parseSubType(TypeDefinition typeDefinition) {
             }
         }
     }
+
+    private HashSet<NativeType> getTypes(TypeDefinition typeDefinition){
```

Comment:
The return type should be just Set<NativeType>, or we expose implementation details.